modules/core/src/main/scala/mapping.scala

```diff
@@ -255,8 +263,20 @@ abstract class Mapping[F[_]] extends QueryExecutor[F, Json] {
   /**
     * Abstract type of field mappings with effects.
     */
-  abstract class EffectMapping extends FieldMapping {
-    def toRootStream: RootStream
+  trait EffectMapping extends FieldMapping
```

Comment:
Would it seem more natural to call this `EffectField` since it's a `FieldMapping`? We're a little inconsistent on this.

Reply:
Agreed on `EffectField`.